dandavison/delta
# Print a notice when delta panics

* Print a notice when delta panics

Setting `DELTA_DEBUG_LOGFILE=crash.log` and repeating the command
writes all information needed to reproduce the crash into that file.

* Add --test-panic

To test the debug handler

----

The notice reads:

```
delta panicked and crashed, sorry :(
To quickly repeat the previous command without delta, run 'export GIT_PAGER=less' first. If you want
to report the crash and it is easy to repeat, do so after 'export DELTA_DEBUG_LOGFILE=crash.log'
then submit the logfile to github at <https://github.com/dandavison/delta/issues>.

!! Make sure there is NO sensitive information in the log file, it will contain the entire diff !!
```
